platformio.ini

```diff
@@ -805,3 +806,21 @@ lib_deps =
   ${esp32.lib_deps}
   TFT_eSPI @ ^2.3.70
 board_build.partitions = ${esp32.default_partitions}
+
+# ------------------------------------------------------------------------------
+# Arduino® Nano ESP32
```

Comment:
just one last (tiny) request, as I would like to keep the file "pure ASCII".
Please change "Arduino®" to "Arduino(R)"

Reply:
Done.